Find and store indexed settlement solution UID (#3353)
cowprotocol/services: database/README.md

---------------|--------|----------|--------
 block\_number | bigint | not null | block in which the settlement happened
 log\_index    | bigint | not null | index in which the event was emitted

>  solver        | bytea  | nullable | public address of the executing solver

Author: Updated by mistake. Thanks.

 tx\_hash      | bytea  | not null | transaction hash in which the settlement got executed

>  tx\_from      | bytea  | nullable | address that submitted the transaction (same as `solver`)

Reviewer: This should never be null since every tx has a `from`.
```suggestion
 tx\_from      | bytea  | not null | EOA that submitted the transaction
```

Author: `tx_from` and `tx_nonce` do not get populated since https://github.com/cowprotocol/services/pull/2283
